lms/templates/emails/reverification_processed_subject.txt

```diff
@@ -0,0 +1,3 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+${_("Re-verification Status")}
```

Comment:
Do we need a full mako template for this?  It seems like this is small enough to include in-line with the function that sends the email.